runtests.sh

```diff
@@ -582,9 +602,9 @@ then
 
     if [ $doRuffFix = true ]
     then
-        ruff check --fix "$(pwd)"
+        ruff check --fix "$homedir"
```

Comment:
it seems the ruff config is ignored https://github.com/Project-MONAI/MONAI/blob/825021d46d6c1a97d6c38bcbfbb0e5e0e148995e/pyproject.toml#L39 if it's not current directory

Reply:
I copied "pyproject.toml" and "setup.cfg" to the specified dir. Does it look good?

Reply:
those files contain specific info about this project codebase, I don't think it should overwrite the specified dir. if it's not easy to fix it, perhaps let's not support ruff for the external dir for now.

Reply:
But there are also other configs, such as flake8, mypy contained in setup.cfg.
https://github.com/Project-MONAI/MONAI/blob/825021d46d6c1a97d6c38bcbfbb0e5e0e148995e/setup.cfg#L163

Reply:
this doesn't seem to related to flake8 or mypy in my understanding, the currently enabled autofix tools are
```
        --formatfix)
            doIsortFix=true
            doBlackFix=true
            doRuffFix=true
            doIsortFormat=true
            doBlackFormat=true
            doRuffFormat=true
```

Reply:
Yes, my original thought is that we can also do something like `./runtests.sh --flake8 --path /my/bundle/scripts`. So if we only support formatfix with path, that makes sense. I will remove the ruff format for now.